### PR TITLE
PLAT-11650-React Native Paper as part of the scaffold

### PR DESCRIPTION
**Pull request by scartag**
## Ticket

PLAT-11650

## Type of PR

- [ ] Bugfix
- [x] New feature
- [ ] Minor changes

Did you make changes to the scaffold?

- [x] Yes, and I have read the [scaffold checklist](/docs/scaffold-checklist.md) document and followed the instructions.
- [ ] No.

Did you make changes to modules or created a new module?

- [ ] Yes, and have read the [modules checklist](/docs/modules-checklist.md) document and followed the instructions.
- [ ] No.

## Changes introduced
React Native Paper in the scaffold

### CODE REVIEW

**Comment by danielsousaio** [comment]
@scartag please go through the [Scaffold checklist document](https://crowdbotics.github.io/modules/scaffold-checklist.html) and make sure to include the manifest file in the changes. Let me know if you have any question.

File: scaffold/template/custom/webpack.config.js [hunk 1]
```diff
@@ -35,103 +35,117 @@ const babelOptions = require("./babel.config.js")
...
-    options: babelOptions
-  }
+    test: /\.js$/,
+    // Add every directory that needs to be compiled by Babel during the build.
+    include: [
+        path.resolve(appDirectory, "index.js"),
+        path.resolve(appDirectory, "App.js"),
+        path.resolve(appDirectory, "modules"),
+        path.resolve(appDirectory, "screens"),
+        path.resolve(appDirectory, "options"),
```
Comment (alaisgomes) [inline scaffold/template/custom/webpack.config.js:70]: Looks like this file is adding 4-space identation. I don't think this is correct for this repo?

File: dist/cookie/{{cookiecutter.project_slug}}/package.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
     "react-native-vector-icons": "^9.2.0",
     "react-native-web": "^0.18.10",
     "react-redux": "^8.0.5",
-    "redux": "^4.2.1"
+    "redux": "^4.2.1",
+    "react-native-paper": "^4.*"
```
Comment (alaisgomes) [inline dist/cookie/{{cookiecutter.project_slug}}/package.json:37]: Can you be a bit more specific with the version? 
I guess a better question is: was there any issues testing the latest react native paper with the current scaffold?
I remember there was an issue with the previous scaffold version, that's why the SPIKE investigated using this one, but there was a comment there saying that we should likely try to get it working with the latest, if possible. Basically just reinstall in your demo app the latest Paper version and check how it runs

If there are some errors, let me know

**Review by alaisgomes — COMMENTED** [review]
Some minor comments

File: dist/cookie/{{cookiecutter.project_slug}}/package.json (hunk 2 above)
Comment (scartag) [reply dist/cookie/{{cookiecutter.project_slug}}/package.json]: Done.

**Review by alaisgomes — APPROVED** [review]

**Review by danielsousaio — APPROVED** [review]
Fixed the `config.js` `previousVersion` to be the previous version SHA, rebased onto master, and ran `yarn run bootstrap`.

**Pull request merged**